week-1/implement/max.js

```diff
@@ -0,0 +1,18 @@
+function max(arr) {
+  let maxArray = -Infinity;
+
+  for (let i = 0; i < arr.length; i++) {
+    if (typeof arr[i] === "number" && arr[i] !== "number") {
```

Comment:
You don't need the expression `arr[i] !== "number"` in your condition the first part of the condition will be enough. plus `arr[i] !== "number"` always going to return true if you test it.